src/components/flowchart/flowchart.js

```diff
@@ -577,6 +633,7 @@ export class FlowChart extends Component {
     if (this.props.slicedPipeline && !isSliceButtonClicked) {
       this.props.onResetSlicePipeline();
       this.updateSlicedPipelineState(null, null, []);
+      this.setState({ showSlicingNotification: false }); // Hide notification when clicking away
```

Comment:
do you think the `this.setState({ showSlicingNotification: false })` can be a part of `this.props.onResetSlicePipeline();`? I can imagine every time you resetPipeline you will need to clear the notification 🤔

Reply:
Thats a good point, I think that should be fine to do!